tests/ci_tests/utils/autopilot_deployment.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from tests.ci_tests.utils.parameters import aws_params
+
+INSTANCE_TYPE = "ml.m5.large"
```

Comment:
what would be the smallest instance we can use?

Reply:
replaced with _ml.m3.medium_, the smallest one